app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,11 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
```

Comment:
it can be its own method, but then it cannot be easily cached because it is a boolean. I.e.

```ruby
def ascii?
  @_ascii ||= bundle.include?(:ascii)
end
```
is not memoizable written that way (maybe there's a better way using `defined`).